src/Chapter2.hs

```diff
@@ -336,7 +336,11 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ ____ [] = []
+subList start end (x:xs)
+  | start > end || start < 0 = []
+  | start == 0 = x : (subList 0 (end - 1) xs)
+  | otherwise = subList (start - 1) (end - 1) xs
```

Comment:
You implemented a correct recursive solution 👍🏻 
Though, in this exercise, you can use standard functions `take` and `drop` to implement `subList` in a more concise way

Reply:
Thanks for the tip!